runtime/gc_vlhgc/IncrementalGenerationalGC.cpp

```diff
@@ -1010,6 +1010,7 @@ MM_IncrementalGenerationalGC::partialGarbageCollectPostWork(MM_EnvironmentVLHGC
 
 	incrementRegionAges(env, _taxationThreshold, true);
 
+	verifyHeapSizing(env);
```

Comment:
I am not insisting, but I don't see a point to have this helper (contains single assertion only). In the case of triggering C-stack is required to figure out which of two places was problematic. I think using assertion directly is more informative.